tests/src/org.ehcache/ehcache/3.10.8-jakarta/build.gradle

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+
+import org.graalvm.internal.tck.TestUtils
+
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = TestUtils.testedLibraryVersion.split("-")[0]
+
+dependencies {
+    testImplementation("org.ehcache:ehcache:$libraryVersion") {
+        capabilities {
+            requireCapability('org.ehcache:ehcache-jakarta')
+        }
+    }
```

Comment:
- I don't know at all whether this can be used normally when the user uses the jakarta variant of ehcache. At least I solved the error of `./gradlew clean test -Pcoordinates=org.ehcache:ehcache:3.10.8-jakarta` through this processing. 
- Can a friend expand https://github.com/oracle/graalvm-reachability-metadata/blob/master/CONTRIBUTING.md ?